nevergrad/optimization/optimizerlib.py

```diff
@@ -2131,6 +2131,12 @@ def _select_optimizer_cls(self) -> base.OptCls:
 
         return optimClass
 
+    def _num_objectives_callback(self) -> None:
+        if self.num_objectives > 1:
+            if self.noise_from_instrumentation or not self.has_noise:
+                # override at runtime
+                self._optim = DE(self.parametrization, self.budget, self.num_workers)
```

Comment:
isnt TwoPointsDE better most of the times?

Reply:
For the multiobjective setting you know better than me...

Reply:
Well no, all DE variants will have the same selection process anyway, so it comes down to which DE variant we would prefer.